bin/ci/before_install.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -v
 
+sudo sed -i 's/127.0.1.1/127.0.0.1/' /etc/hosts
```

Comment:
If this is to fix [this spec](https://github.com/ManageIQ/manageiq/blob/master/spec/lib/miq_environment_spec.rb#L17-L19) then I believe that is a flawed test.  It is using `Socket` to find the loopback address but then using `hostname -i` which depends on name resolution to find the loopback address.  This fails if you have localhost and a hostname with a registered loopback address (as I do as well)

```
$ cat /etc/hosts
127.0.0.1	localhost
127.0.1.1	desktop.example.com	desktop

$ hostname -i
127.0.1.1
```
Since `hostname -i` is using `desktop` it finds `127.0.1.1` instead of `127.0.0.1`.  A better course of action would be to fix that spec so that it works properly.

Reply:
Looks like this code was updated in this [PR](https://github.com/ManageIQ/manageiq/pull/20341).  /cc @djberg96

Reply:
Oops, guess my test was bad. @agrare, recommendations?

Reply:
Put in https://github.com/ManageIQ/manageiq/pull/20992, we can discuss over there

Reply:
@odidev #20992 is merged, so can you try again without this line?

Reply:
Do you thing that `sudo: required` can be removed from all of the matrix definitions as well?  I would think that the line above was the only reason it was included.

Reply:
Oh yes good call...the `sudo: required` should be removed.